### PR TITLE
Merge develop into SDL RC

### PR DESCRIPTION
**Pull request by LevchenkoS**
@okozlovlux, @LuxoftAKutsan, please, review.

### CODE REVIEW

File: src/3rd_party-static/test/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015, Ford Motor Company
+# Copyright (c) 2016, Ford Motor Company
```
Comment (okozlovlux) [inline src/3rd_party-static/test/CMakeLists.txt]: @LevchenkoS , 2016?

**Comment by LevchenkoS** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by LuxoftAKutsan** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Pull request merged**